yoti-connect/yoti-connect.php

```diff
@@ -139,7 +139,7 @@ function yoti_connect_login($user_login=null, $user=null)
     }
 
     $activityDetails = YotiConnectHelper::getYotiUserFromStore();
-    if ($activityDetails && empty($_SESSION['yoti_nolink']))
+    if ($activityDetails && (!isset($_POST['yoti_nolink']) || empty($_POST['yoti_nolink'])))
```

Comment:
Do you think it would be a good idea to set a variable with the value `(!isset($_POST['yoti_nolink']) || empty($_POST['yoti_nolink'])`?